dbms/src/Storages/DeltaMerge/RowKeyFilter.h

```diff
@@ -150,7 +151,7 @@ inline Block filterUnsorted(const RowKeyRanges & rowkey_ranges, Block && block,
 
     for (size_t i = 0; i < block.columns(); ++i)
     {
-        auto & column = block.getByPosition(i);
+        auto & column = block.safeGetByPosition(i);
```

Comment:
`i` must be a valid number in this loop, we can use `getByPosition` to make it run faster